### PR TITLE
Refactor send transaction example

### PR DESCRIPTION
**Pull request by Szymx95**

### CODE REVIEW

File: packages/example/src/components/Transactions/TransactionForm.tsx [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+const getTransactionErrorMessage = (transaction: TransactionStatus) => {
+  if ('errorMessage' in transaction) return transaction.errorMessage
+  else return ''
+}
+
+const getTransactionChainId = (transaction: TransactionStatus) => {
+  if ('chainId' in transaction) return transaction.chainId
+  else return undefined
+}
```
Comment (marekkirejczyk) [inline packages/example/src/components/Transactions/TransactionForm.tsx:53]: No need to extract helper, just use `transaction?.chainId`

File: packages/example/src/components/Transactions/TransactionForm.tsx [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+    <div style={{ color: color, textAlign: 'center' }}>{text}</div>
+  </InformationRow>
+)
+
+const transactionErrored = (transaction: TransactionStatus) => {
+  return 'errorMessage' in transaction
+}
+
+const getTransactionErrorMessage = (transaction: TransactionStatus) => {
+  if ('errorMessage' in transaction) return transaction.errorMessage
```
Comment (marekkirejczyk) [inline packages/example/src/components/Transactions/TransactionForm.tsx:46]: No need to extract helper, just use `transaction?.errorMessage`

Comment (marekkirejczyk) [reply packages/example/src/components/Transactions/TransactionForm.tsx:46]: See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html

Comment (nezouse) [reply packages/example/src/components/Transactions/TransactionForm.tsx:46]: It's correct this way, TranscationStatus without checks is just 
```js
{
      status: string
}
```

Comment (marekkirejczyk) [reply packages/example/src/components/Transactions/TransactionForm.tsx:46]: Fair enough :)
I was wondering if we can do:
```ts
transactionErrored(transaction) && transaction?.errorMessage
```
Does not seem awesome.

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**